### PR TITLE
Challenge 2/api

### PR DESCRIPTION
**Pull request by kyakusahmed**

### CODE REVIEW

File: app/views.py [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+
+@app.route('/', methods=['GET'])
+def index():
+    """opening route."""
+    return jsonify({
+        "status": 200, 'message': 'welcome to Epic Email.'
+        }), 200
+
+
+@app.route('/api/v1/users/signup', methods=['POST'])
```
Comment (mendozabree) [inline app/views.py]: Please consider changing the URLs for all your endpoints to match those in the document

File: app/email.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+            "lastname": lastname,
+            "password": password
+        }
+        self.users.append(user)
+        return user
+
+    def send_email(self, subject, message, status, sender_id, receiver_id):
+        """add new email."""
+        email = {
+            "email_id":  len(self.emails)+1,
```
Comment (mendozabree) [inline app/email.py]: Please be consistent in naming your variables.
Check out this [resource](https://medium.com/@dasagrivamanu/python-naming-conventions-the-10-points-you-should-know-149a9aa9f8c7)

**Review by mendozabree — CHANGES_REQUESTED** [review]
@kyakusahmed good progress!
There a couple of pep8 issues. Install autopep8 and flake8 to help you see and fix the errors.
Also reach out to Jenny, to show you how to structure your files better

File: app/views.py (hunk 1 above)
Comment (kyakusahmed) [reply app/views.py]: changed all the Urls

**Review by mendozabree — APPROVED** [review]
LGTM

**Pull request merged**